### PR TITLE
Phone case variants, to infinity, and beyond.

### PR DESCRIPTION
**Pull request by DoctorBoomstick**
#### Summary
Content "adds multiple boatloads of new phone case variants, based, mostly, around different horror movies."
#### Purpose of change
Creating this PR on behalf of Cosmic on the discord, with 99% of the variants here being totally their work. Variants are fun, cool, and interesting, and this PR simply adds more for the sake of building up those numbers.
#### Describe the solution
This simply adds a whole cinema’s worth of horror-themed phone cases to the already existing variants, with all themed cases balanced to have a much lower spawn chance than their plainly colored brethren.
I placed this PR as a draft as both I and Cosmic are minorly iffy on the acceptability of including name-dropping content within CDDA, lest the armies of copyright lawyers be heard on the march, and figured it would be prudent to place the changes as a draft to see the general consensus and if changes ought to be made.
#### Describe alternatives you've considered
Going to bed rather than including Xenomorph phone cases in cdda. Unacceptable.
#### Testing
They simply slot into the existing variants structure, and no syntax errors were reported in the process of linting.
#### Additional context
See, almost every single horror movie created for context. It might take a while.

### CODE REVIEW

File: data/json/items/containers.json [hunk 1]
```diff
@@ -2250,43 +2250,565 @@
...
+      {
+        "id": "matt_case",
+        "name": { "str": "Maniac Cop smartphone case" },
+        "description": "The design of this waterproof smartphone case is inspired by the iconic police uniform of the maniac cop.  The case features the police badge, symbolizing authority, with the maniac's glove-clad hand over it in a steel-like color against a dark blue background.  It also includes a bloody police baton graphic to further emphasize the theme.",
+        "weight": 5
+      },
+      {
+        "id": "herbet_case",
+        "name": { "str": "Re-Animator smartphone case" },
+        "description": "Inspired by Dr Herbert West's reanimation serum and the undead creatures it creates from the 'Re-Animator' horror movie series,  This case features a medical syringe graphic with a glowing, green liquid radiating outward, symbolizing the serum substance, against a plain black background.  Eh, ironic.",
```
Comment (TheShadowFerret) [inline data/json/items/containers.json:2464]:
```suggestion
        "description": "Inspired by Dr. Herbert West's reanimation serum and the undead creatures it creates from the 'Re-Animator' horror movie series,  This case features a medical syringe graphic with a glowing, green liquid radiating outward, symbolizing the serum substance, against a plain black background.  Eh, ironic.",
```

File: data/json/items/containers.json [hunk 2]
```diff
@@ -2250,43 +2250,565 @@
...
+      },
+      {
+        "id": "grudge_case",
+        "name": { "str": "The Grudge smartphone case" },
+        "description": "This waterproof smartphone case features the haunting Grudge ghost's white face on a black background, with her multiple horror contortions through the movie sparsely hinted at in a subtle way around her face.",
+        "weight": 5
+      },
+      {
+        "id": "rec_case",
+        "name": { "str": "[R E C ] smartphone case" },
```
Comment (TheShadowFerret) [inline data/json/items/containers.json:2577]:
```suggestion
        "name": { "str": "[R E C] smartphone case" },
```

**Comment by Procyonae** [comment]
Why have you removed waterproof from the names?

**Comment by DoctorBoomstick** [comment]
> Why have you removed waterproof from the names?

I felt like the names became way, way, way too wordy and giant on most of them when “waterproof” was included in the name. we don’t have any non-water proof cases currently in game, so mentioning that element in the name is, in my view, rather redundant. Plus, from what I’ve read as I was doing some grammer checks on them, most of the descriptions state that they’re water proof anyhow.

**Comment by Procyonae** [comment]
> I felt like the names became way, way, way too wordy and giant on most of them when “waterproof” was included in the name. we don’t have any non-water proof cases currently in game, so mentioning that element in the name is, in my view, rather redundant. Plus, from what I’ve read as I was doing some grammer checks on them, most of the descriptions state that they’re water proof anyhow.

Fair enough

File: data/json/items/containers.json [hunk 3]
```diff
@@ -2250,43 +2250,565 @@
...
+      {
+        "id": "candyman_case",
+        "name": { "str": "Candyman smartphone case" },
+        "description": "This design is inspired by Candyman from the 'Candyman' horror movie series.  With his bee-themed terror, this case features a honeycomb pattern background in a dark yellowish-brownish hue, complemented with bees in a similar color scheme.  The Candyman's face is, alongside his hook, in black and blue tones, complementing the honeycomb pattern.",
+        "weight": 5
+      },
+      {
+        "id": "saw_case",
+        "name": { "str": "Saw smartphone case" },
+        "description": "This design is inspired by the infamous 'reverse bear trap' from Saw.  This case features an in-your-face graphic of the trap with a rusted metallic texture. A blood spattering effect nearby is also included, probably to add extra horror detail.",
```
Comment (Procyonae) [inline data/json/items/containers.json:2428]:
```suggestion
        "description": "This design is inspired by the infamous 'reverse bear trap' from Saw.  This case features an in-your-face graphic of the trap with a rusted metallic texture.  A blood spattering effect nearby is also included, probably to add extra horror detail.",
```

File: data/json/items/containers.json [hunk 4]
```diff
@@ -2250,43 +2250,565 @@
...
+      {
+        "id": "matt_case",
+        "name": { "str": "Maniac Cop smartphone case" },
+        "description": "The design of this waterproof smartphone case is inspired by the iconic police uniform of the maniac cop.  The case features the police badge, symbolizing authority, with the maniac's glove-clad hand over it in a steel-like color against a dark blue background.  It also includes a bloody police baton graphic to further emphasize the theme.",
+        "weight": 5
+      },
+      {
+        "id": "herbet_case",
+        "name": { "str": "Re-Animator smartphone case" },
+        "description": "Inspired by Dr. Herbert West's reanimation serum and the undead creatures it creates from the 'Re-Animator' horror movie series,  This case features a medical syringe graphic with a glowing, green liquid radiating outward, symbolizing the serum substance, against a plain black background.  Eh, ironic.",
```
Comment (Procyonae) [inline data/json/items/containers.json:2464]:
```suggestion
        "description": "Inspired by Dr. Herbert West's reanimation serum and the undead creatures it creates from the 'Re-Animator' horror movie series, this case features a medical syringe graphic with a glowing, green liquid radiating outward, symbolizing the serum substance, against a plain black background.  Eh, ironic.",
```

File: data/json/items/containers.json [hunk 5]
```diff
@@ -2250,43 +2250,565 @@
...
+      {
+        "id": "coffinjoe_case",
+        "name": { "str": "Coffin Joe smartphone case" },
+        "description": "This design is inspired by the Brazilian horror icon, Coffin Joe (Zé do Caixo).  It has the silhouette of him wearing his classic top hat and suit, along with the symbolic scythe, against a purple backdrop in the background.",
+        "weight": 5
+      },
+      {
+        "id": "xenomorph_case",
+        "name": { "str": "Alien smartphone case" },
+        "description": "This design is inspired by the Xenomorphs' threat and the spaceship's doomed crew. This case features a graphic of Ripley with the Alien Queen's tail coiling around the phone case, with a striking color combination of dark blues and greens to create an eerie mood.",
```
Comment (Procyonae) [inline data/json/items/containers.json:2494]:
```suggestion
        "description": "This design is inspired by the Xenomorphs' threat and the spaceship's doomed crew.  This case features a graphic of Ripley with the Alien Queen's tail coiling around the phone case, with a striking color combination of dark blues and greens to create an eerie mood.",
```

**Comment by Maleclypse** [comment]
Is this ready for review?
